docker/sui-node/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:1.60.0 AS chef
+FROM rust:latest AS chef
```

Comment:
We intentionally make rust version upgrades manual. I like latest for myself, but the project may not be tested to match.
😅